cmd/ibctest/ibctest_test.go

```diff
@@ -55,8 +55,8 @@ func setUpTestMatrix() error {
 		testMatrix.Relayers = []string{"rly"}
 		testMatrix.ChainSets = [][]ibctest.BuiltinChainFactoryEntry{
 			{
-				{Name: "gaia", Version: "latest", ChainID: "cosmoshub-1004", NumValidators: 1, NumFullNodes: 1},
-				{Name: "osmosis", Version: "latest", ChainID: "osmosis-1001", NumValidators: 1, NumFullNodes: 1},
+				{Name: "gaia", Version: "v7.0.1", ChainID: "cosmoshub-1004", NumValidators: 2, NumFullNodes: 1},
+				{Name: "osmosis", Version: "v7.2.0", ChainID: "osmosis-1001", NumValidators: 2, NumFullNodes: 1},
```

Comment:
This was the quickest path to ensure CI stability.

I also feel if a new user runs ibctest binary, they'd probably also want a stable experience.

The tradeoff is remembering to update these versions... Could be ldflags or similar. 

An alternative (which requires more work) is to pass in a matrix file for CI.

Reply:
Fine by me for CI/new user stability. I'm sure we will have our full blown matrix file(s) for the full suite output soon enough.